AmazonChimeSDK/AmazonChimeSDK/audiovideo/data/transcript/TranscriptionStatus.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  TranscriptionStatus.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+@objcMembers public class TranscriptionStatus: NSObject, TranscriptEvent {
+    public let type: TranscriptionStatusType
+    public let eventTimeMs: Int64
+    public let transcriptionRegion: String
+    public let transcriptionConfiguration: String
```

Comment:
Why is configuration string?

Reply:
One alternative is to explicitly define strongly typed/nested classes but we felt it's just overkill since builders are likely going to use this for logging purposes. It also provides more flexibility if we want to add new fields to the configuration with a plain string. That's why JS SDK also has it as string: https://github.com/aws/amazon-chime-sdk-js/blob/master/src/transcript/TranscriptionStatus.ts#L10